testing-modules/junit-5/src/test/java/com/baeldung/execution/time/RandomNumberUnitTest.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.execution.time;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class RandomNumberUnitTest {
+
+    @Test
+    public void whenTwoRandomNumberThenAssertUniqueness() {
+
+        Assertions.assertNotEquals(Math.random(), Math.random());
```

Comment:
We usually would statically import `Assertions` rather than prefix the assertion method with it.